packages/design-system/src/components/Icon/helpers.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export function generateIcon(icon, iconSize, iconColor) {
```

Comment:
if it's that simple maybe we could directly use this `React.createElement` in the component instead of some mysterious generateIcon

Reply:
Sure, we can. I wanted to extract it to helpers to have less code in Icon file, but indeed it's a short function